flow/controllers/velocity_controllers.py

```diff
@@ -184,7 +172,7 @@ class PISaturation(BaseController):
 
     def __init__(self, veh_id, car_following_params):
         """Instantiate PISaturation."""
-        BaseController.__init__(self, veh_id, car_following_params, delay=1.0)
+        BaseController.__init__(self, veh_id, car_following_params, delay=0.0)
```

Comment:
No clue why the default followerstopper has a delay of 1.0. that seems definitely incorrect.

Reply:
Think it doesn't actually do anything anymore. But, good point,